fhir-persistence-schema/README.md

```diff
@@ -141,6 +141,23 @@ Edit `wlp/usr/servers/fhir-server/config/default/fhir-server-config.json` and ad
 --tenant-key "<the-base64-tenant-key>"
 ```
 
+### Add a Key to Existing Tenant
+To add a tenant key for an existing tenant, replace FHIRDATA with your client schema, and TENANT_NAME with your tenant's name. 
+
```

Comment:
do we have info about when is needed to add a key to existing tenant? and how to delete the old tenant key if need etc.

Reply:
I actually had the same thought.  I don't think there is a function to remove a prior key, but I think there should be.

Reply:
I can add a comment on this.

Reply:
![image](https://user-images.githubusercontent.com/3016328/77464143-1eca5d00-6ddd-11ea-9693-596b64c0db68.png)
We only have allocate and drop and add

Reply:
thoughts?

Reply:
allocateTenant will add the tenant and also create a tenant key. 
does dropTenant drop all tenant keys? I guess it should. 
did Robin add the add key for existing tenant feature for security reason? e.g, the tenant key is expose, so need a new key for the tenant ... if so, I think, we do need a command to remove the key ...  and I guess, all the tenant keys of a tenant can work to access the tenant, so if don't delete the old key could bring security issue. ...

Reply:
or it could be that, Robin added this feature just in case the old tenant key is missed/forgotten, but we still want to access that tenant, so this make it handy for us to just create a new key for the tenant and then use it to access the tenant ...

Reply:
dropTenant does not remove a key.  it just change the tenant status and removes the partition.  It was for key rotation (except there are NO delete-key features)

Reply:
ha, I see. then I think if possible, we need:
(1) add removing all the tenant keys function in dropTenant, otherwise they are just useless garbage data after the tenant is dropped.
(2) add remove key command, use tenant name and key as parameter.
And as I remember, the tenant key itself is not stored in DB, but the hash with a salt of the key is stored in db instead, so we will need to remove the hash from the table instead...

Reply:
but anyway, I'm OK with this document change.

Reply:
it's best to leave it.  I think it's good audit data, and ensures the id is not re-used.

Reply:
I think remove key should be added, that's fairly easy to use.  If agreed, we can create an issue to track